tests/main/views/test_brief_response.py

```diff
@@ -252,6 +261,20 @@ def test_cannot_create_brief_response_when_supplier_isnt_eligible(self, live_dos
         assert res.status_code == 400
         assert 'Supplier is not eligible to apply to this brief' in res.get_data(as_text=True)
 
+    def test_cannot_create_a_brief_response_if_framework_status_is_not_live_or_expired(self, expired_dos_framework):
+        for framework_status in ['coming', 'open', 'pending', 'standstill']:
+            with self.app.app_context():
+                db.session.execute(
+                    "UPDATE frameworks SET status=:status WHERE slug='digital-outcomes-and-specialists'",
```

Comment:
I remember this sort of thing being talked about in Ben's talk regarding changing our database state and not rolling it back mid unit test. I believe the plan was to move to some package that would help with this... Can't remember exactly what the outcome was and whether we now advised against changing the frameworks or we should try and do teardowns...

Another option might be to target `expired_dos_framework` by ID rather than slug so you know you are only changing one of our fixtures rather than the real DOS framework in the DB. 

I don't remember the exacts but could be worth having a quick chat with Ben if needbe.